src/MessageUI/MessageUI.cs

```diff
@@ -12,22 +12,19 @@
 
 namespace XamCore.MessageUI {
 
-	// untyped enum -> MFMailComposeViewController.h
```

Comment:
If they were changed to `NSInteger` then it's a breaking change and we'll need to review the API using them (as we can't change the public `enum` size).

The comment will need to be updated, but not just removed.